src/Nfield.Quota/QuotaVariableDefinitionCollection.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+using Nfield.Quota.Helpers;
+
+namespace Nfield.Quota
+{
+    public class QuotaVariableDefinitionCollection : List<QuotaVariableDefinition>
+    {
+        public ICollection<QuotaVariableDefinition> Variables { get; }
```

Comment:
I don't think you want this, it seems a bit double since you're already deriving from it.